configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
@jart The challenge here is that libjpeg-turbo is an assembly-based code base, rather than pure C. This means that the configuration for it is pretty tricky, and the default build for libjpeg is based on autotools (autoconf, automake) as well as nasm. This makes it messy because suddenly for one library substitution, the build grows from Bazel to 3 additional build tools. If the lib substitution is optional, users who don't care about the improved performance can keep building like they always have, just with Bazel. Users who might benefit from the extra performance and don't mind installing extra tools could do so, but I think forcing extra tools on every user would be painful. The other issue here is that as far as I know, doing assembly builds by themselves directly in Bazel would also be non-trivial, hence I'm reusing the build definitions from libjpeg-turbo's code base.